dev-scripts/cc-router-debug-docker-image/build-cc-router-debug-docker-image.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Let's add the license header

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2998/commits/2ddcaf432f8e7954fb17193c8353a2a30a1e8e9b